gpkit/solution_array.py

```diff
@@ -80,17 +80,7 @@ def sens(self, nomial):
 
```

Comment:
(I'm referring to the `sens` method -- looks like this inline comment is showing up in an unexpected place)

Then should `sens(self, nomial)` change to `sens(self, key)` and the docstring be updated to remove the part about signomial input?